native/core/Cargo.toml

```diff
@@ -77,6 +77,7 @@ jni = { version = "0.21", features = ["invocation"] }
 lazy_static = "1.4"
 assertables = "7"
 hex = "0.4.3"
+datafusion-functions-nested = "46.0.0"
```

Comment:
can we make it to `workspace = true` so that we do not forget to update here when we upgrade DF

Reply:
I'm not sure if it is possible to make it `workspace = true`, this package is optional for test only and optional packages are not allowed on workspace level. 

I can include it in workspace as non optional but it will be compiled into the target binary which is probably not expected